packages/nextjs/src/client-boundary/uiComponents.tsx

```diff
@@ -35,12 +37,14 @@ export const SignIn = (props: SignInProps) => {
 
 export const SignUp = (props: SignUpProps) => {
   const { signUpUrl: repoLevelSignUpUrl } = useClerkNextOptions();
+  const { routing, path, ...restProps } = props;
```

Comment:
🙃 `const { routing: _routing, path: _path, ...restProps } = props;`  (probably there is an unused variable eslint warning here that will be resolved with this).

Reply:
Variables starting with `_` will be ignored from the `no-unused-vars` rule in ESLint